.github/workflows/lagoon-tests.yml

```diff
@@ -0,0 +1,16 @@
+---
+name: "Tests of deployment"
+
+on:
+  deployment_status
+
+# @todo - this action does nothing useful right now, but it is to show that
+# the site URL is available as part of deployment_status.
```

Comment:
I appreciate the comment but I suggest removing this file entirely.

As I see it we should not run tests on the deployed environment as implied in the workflow name. We do this on the corresponding environment established in CI.